flag_engine/projects/schemas.py

```diff
@@ -11,6 +11,7 @@ class BaseProjectSchema(Schema):
     name = fields.Str()
     organisation = fields.Nested(OrganisationSchema)
     hide_disabled_flags = fields.Bool()
+    enable_realtime_updates = fields.Bool()
```

Comment:
Do we need to worry about the fact that it won't currently exist? I'm not sure if we perhaps need to add `required=False, default=False` or something?

Reply:
Required is by default `false`. I have added the default to model since that is what we use to process stuff

Reply:
Understood. Just wanted to check, thanks.